README.md

```diff
@@ -3,81 +3,174 @@
+- **Step 5.** Connect to your instrument using one of these methods:
+   - Use the Instrument Explorer to discover instruments on your network and connect with a right-click
+   - Click the "+" button in the Instrument Explorer to connect via IP address or VISA resource string
+   - Use the Command Palette (Ctrl+Shift+P) and run "TSP: Connect"
+- **Step 6.** [Configure your project](#configure-your-project) for your [TSP-Link™][tsp-link] instrument configuration.
+- **Step 7.** Edit and run your TSP scripts by right-clicking them in the file explorer,
+              file tabs, or editor window and selecting "Send Script to Terminal"
+
+    ![Send Script to Terminal][pic-send-script-to-terminal]
 
 ## Demo Video
 
 <a href="https://www.tek.com/en/video/product-demo/leveraging-test-script-processor-technology-with-keithley-tsp-toolkit"><img width="560" height="315" src="./resources/DemoVideoThumbnail.png" altText="TSP Toolkit Demo Video"></img></a>
 
-## Resources and Tutorials
 
-- [TSP Toolkit Feature Walkthrough][tsp-toolkit-feature-walkthrough]
-- [TSP Landing Page on Tek.com][tsp]
-- [TSP Video Series][tsp-video-series]
-- [App Note: How to Write TSP Scripts for TSP][app-note-how-to-write-tsp-scripts]
-- [TSP Script Example Repository][tsp-script-examples]
+## TSP View Container
 
+The TSP Toolkit provides a dedicated view container in the Activity Bar that offers access to all major features:
 
-## Installed Extensions
+### View Sections
 
-Tektronix TSP Toolkit will automatically install the [sumneko.lua][sumneko]
-extension to use all of the language features it provides.
+- **INSTRUMENTS**: Displays discovered and saved instruments on your network
+  - Connect to instruments
+  - Manage instrument connections
+  - Perform instrument operations (reset, abort, firmware upgrade)
 
-Extensions installed through the marketplace are subject to the [Marketplace Terms of Use][marketplace-tou].
+- **SYSTEM CONFIGURATIONS**: Configure your TSP-Link instrument network
+  - Create new system configurations
+  - Fetch configurations from connected instruments
+  - Switch between different configurations
 
-## Quick Start
+- **SCRIPT GENERATION**: Create and manage TSP scripts without manual coding
+  - Create new script generation sessions
+  - Access saved script configurations
+  - Generate ready-to-use TSP scripts
 
-- **Step 1.** Connect your TSP-enabled Tektronix and Keithley instrument to your local network (LAN).
-- **Step 2.** Install the [Tektronix TSP Toolkit Visual Studio Code Extension][tsp-toolkit-marketplace].
-- **Step 3.** Open or create a folder for your TSP project.
+### Accessing the View
 
-    ![Open Folder][pic-open-folder]
+Click the TSP Toolkit icon <img src="./resources/tsp-toolkit-view-container-icon.svg" alt="TSP Toolkit Icon" width="24" height="24" style="vertical-align:middle;"> in the Activity Bar (usually on the left side of VS Code) to access all these features.
```

Comment:
Apparently npm doesn't like to have SVGs in the README, so you'll need to export a PNG (or use a screenshot).